terraform/modules/emr/templates/emr/configuration.mysql.json

```diff
@@ -0,0 +1,63 @@
+[
+  {
+    "Classification": "core-site",
+    "Properties": {
+      "hadoop.proxyuser.livy.hosts": "*",
+      "hadoop.proxyuser.livy.groups": "*"
+    }
+  },
+  {
+    "Classification": "livy-conf",
+    "Properties": {
+      "livy.spark.deploy-mode": "cluster",
+      "livy.file.local-dir-whitelist": "/",
+      "livy.repl.enable-hive-context": "true",
+      "livy.impersonation.enabled": "true",
+      "livy.spark.yarn.security.credentials.hiveserver2.enabled": "true",
+      "livy.server.port": "8998"
+    }
+  },
+  {
+    "Classification": "yarn-site",
+    "Properties": {
+      "yarn.log-aggregation-enable": "true",
+      "yarn.log-aggregation.retain-seconds": "-1",
+      "yarn.nodemanager.remote-app-log-dir": "${logs_bucket_path}/yarn/"
+    }
+  },
+  {
+    "Classification": "spark",
+    "Properties": {
+      "maximizeResourceAllocation": "false"
+    }
+  },
+  {
+    "Classification": "spark-defaults",
+    "Properties": {
+      "spark.r.shell.command": "/opt/R/R-${r_version}/bin/R",
+      "spark.r.command": "/opt/R/R-${r_version}/bin/Rscript",
+      "spark.sql.catalogImplementation": "hive",
+      "spark.sql.warehouse.dir": "${data_bucket_path}/external",
+      "spark.driver.extraJavaOptions": "-XX:+UseConcMarkSweepGC -XX:CMSInitiatingOccupancyFraction=70 -XX:MaxHeapFreeRatio=70 -XX:+CMSClassUnloadingEnabled -XX:OnOutOfMemoryError='kill -9 %p' -Dhttp.proxyHost='${proxy_host}' -Dhttp.proxyPort='3128' -Dhttp.nonProxyHosts='${full_no_proxy}' -Dhttps.proxyHost='${proxy_host}' -Dhttps.proxyPort='3128'",
+      "spark.executor.extraJavaOptions": "-verbose:gc -XX:+PrintGCDetails -XX:+PrintGCDateStamps -XX:+UseConcMarkSweepGC -XX:CMSInitiatingOccupancyFraction=70 -XX:MaxHeapFreeRatio=70 -XX:+CMSClassUnloadingEnabled -XX:OnOutOfMemoryError='kill -9 %p' -Dhttp.proxyHost='${proxy_host}' -Dhttp.proxyPort='3128' -Dhttp.nonProxyHosts='${full_no_proxy}' -Dhttps.proxyHost='${proxy_host}' -Dhttps.proxyPort='3128'"
+    }
+  },
+  {
+    "Classification": "spark-hive-site",
+    "Properties": {
+      "javax.jdo.option.ConnectionURL": "jdbc:mysql://${hive_metastore_endpoint}:3306/${hive_metastore_database_name}?createDatabaseIfNotExist=true",
+      "javax.jdo.option.ConnectionDriverName": "org.mariadb.jdbc.Driver",
+      "javax.jdo.option.ConnectionUserName": "username",
```

Comment:
This supposed to be username=username and password=password?

Reply:
Nope, rebase error, fixing